ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -425,12 +442,10 @@ instance
       , (NodeToClientV_19, CardanoNodeToClientVersion15)
       , (NodeToClientV_20, CardanoNodeToClientVersion16)
       , (NodeToClientV_21, CardanoNodeToClientVersion17)
+      , (NodeToClientV_22, CardanoNodeToClientVersion18)
       ]
 
-  -- This is not set to NodeToClientV_21 on purpose because that one is just a
-  -- stub. Once we have a proper ouroboros-network to integrate that comes with
-  -- said version and we remove the SRP then we can bump this value.
-  latestReleasedNodeVersion _prx = (Just NodeToNodeV_14, Just NodeToClientV_20)
+  latestReleasedNodeVersion _prx = (Just NodeToNodeV_15, Just NodeToClientV_22)
```

Comment:
- Why do we need a new `NodeToNode` version? https://github.com/IntersectMBO/ouroboros-network/blob/8b7b545b028d112e0c09362e7915e6d0ddfe1bb6/ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs#L71-L72 says "SRV support", but isn't this purely N2C-related?

   Or is this just for diagnostics?

 - Why do we need `NodeToClientV_22`? Wouldn't `NodeToClientV_21` suffice given that it hasn't been released yet?

Reply:
* No, it's not just `N2C` feature.  Nodes can be configured with SRV records, and the peer selection can use SRV records registered on the chain.  Technically, we don't need a new `node-to-node` version, but we wanted to mark when SRV support begins.  This might be useful for debugging issues.
* I wasn't aware that it was not released, but I don't think this matters.

Reply:
Then this sounds fine, it is not a huge deal to have technically unnecessary N2N/N2C versions :+1: